### PR TITLE
Node remediation test

### PR DESCRIPTION
**Pull request by cynepco3hahue**
This PR:
- [x] add helper methods
- [x] add node remediation test
- [x] configure repo as part of stdci

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
ci add to whitelist

File: tests/node_remediation_test.go [hunk 1]
```diff
@@ -21,24 +21,111 @@ package tests_test
...
+					if err != nil {
+						return nil
+					}
+					return nr
+				}, 30*time.Second, 1*time.Second,
+			).ShouldNot(BeNil())
+		})
+
+		AfterEach(func() {
+			By("Starting kubelet service on the non-master node")
```
Comment (beekhof) [inline tests/node_remediation_test.go]: Stupid question... why do you need to start it again?

Comment (cynepco3hahue) [reply tests/node_remediation_test.go]: @beekhof It does not really restart the node via IPMI, so `kubelet` service will stay down forever.
To be sure that I will have the same environment for each test I start it again.
I also wanted to check `provider-external` jobs, to see if it succeeds, but I will do it in the separate PR, after we will finish discussion around https://github.com/kubevirt/cluster-api-provider-external/pull/6.

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by danielBelenky** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
We have some issues to fetch images from the docker hub, but it does not relate to the PR, because when fetching works, job succeeds.
For reference https://jenkins.ovirt.org/job/kubevirt_node-recovery_standard-check-pr/9//artifact/check-patch.k8s-1.11.0-devel.el7.x86_64/mock_logs/script/stdout_stderr.log

**Pull request merged**